x/cronos/middleware/conversion_middleware.go

```diff
@@ -165,15 +160,11 @@ func (im IBCConversionModule) OnTimeoutPacket(
 		if err != nil {
 			return err
 		}
-		denom := im.getIbcDenomFromPacketAndData(packet, data)
+		denom := im.getIbcDenomFromDataForRefund(data)
```

Comment:
`getIbcDenomFromPacketAndData` seems more complex than `getIbcDenomFromDataForRefund`, is it equivalent?

Reply:
refund [break](https://github.com/crypto-org-chain/cronos/pull/613#discussion_r937284892) on timeout and ack err, you could trace the previous discuss